receipt/receipt.go

```diff
@@ -0,0 +1,112 @@
+/*
+ * This example showcases the usage of creator templates to create a reciept document
+ *
+ * Run as: go run receipt.go.go
+ */
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+)
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+
+// Field represents field item.
+type Field struct {
+	FieldName  string `json:"FieldName"`
+	FieldValue string `json:"FieldValue"`
+}
+
+// Reciept represent Recipt object.
```

Comment:
`Receipt` instead of `Reciept`. The name of the type is mis-typed as well.

Actually, there are bunch of typos in this file: `reciept` instead of `receipt`. Please change all of them.
The same typo is in the name of the PR as well. Please change that too.

Reply:
Yeah, I see that now. I will update it.